packages/angular_devkit/build_angular/package.json

```diff
@@ -75,7 +75,7 @@
   },
   "peerDependencies": {
     "@angular/compiler-cli": ">=10.0.0-next.0 < 11",
-    "typescript": ">=3.6 < 3.9"
+    "typescript": ">=3.9 < 4.0"
```

Comment:
```suggestion
    "typescript": ">=3.9 < 3.10"
```
Just in case.

Reply:
This is actually the same as the FW.

Reply:
What happens if there is though?  I think it also sends conflicting signals regarding the need to be cautious when upgrading TS minors.

Reply:
Fair point, though that’s very unlikely to happen.

Reply:
Updated